### ISSUE TITLE
v0.23-RC4 can't read Sony A6400 .ARW RAW file metadata

### ISSUE THREAD

**Issue by Ricostyle21** [simulot/immich-go#582]
Hey!
Tried importing JPG and ARW RAW files and wanted the newest RC4 to stack them, but with every ARW file the console outputs the warning: 
`2024-12-27 23:41:30 WRN Info file=DSC04847.ARW warning=can't read metadata for this format '.arw'`
I assume this is also the cause for the issue that nothing in this folder is being stacked, even with the --manage-raw-jpeg option turned on. (Also tried with the stack command, same issue).

EDIT: it does seem to end up stacked in Immich (although it wasn't immediately after importing, but the logs also don't mention any stacking)??

**Comment by simulot** [comment]
You're right. Stacking is based on file name and date of capture. If I can't get the date of capture, I don't try to stack them.

I'm not aware about ARW files, therefore my "own" metadata extractor just ignores them.
Internet says this should be easy to find EXIF data... I'll give a try. You may share some file samples.

However, the RC4 comes with the stack function that relies on immich data instead of my own reading procedure. This command accepts all stack options, and may work out of the box.

**Comment by simulot** [comment]
I got the files. Thanks. Nice city, always a pleasure to come to Lisbon.

I didn't pay too much attention to MKV, as they aren't the main target of immich systems
